QUESTIONS.md

```diff
@@ -1,13 +1,6 @@
 # Questions about Code, Node, Express, React
 
-
-What allows for the front end to connect with the back end ?
-
-What is a wiggle sort and when would it matter?
```

Comment:
This is a kind of sorting, but I don't think we have come across this in our trainings.  Based on discussion with my team I got to know there are some you tube videos that helps explain this.